Added docs for Permanently deleting a drive item. (#21780)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/driveitem-permanentDelete.md

> ---

Reviewer: Filenames must be in lowercase.

Author: Done. Thanks

author: mahgoyal
ms.date: 06/28/2023

> title: Permanently delete a file or folder

Reviewer: ```suggestion
title: Permanently delete a file or folder.
```

ms.localizationpriority: medium
ms.prod: "sharepoint"

> description: "Permanently delete a DriveItem by using its ID."

Reviewer: ```suggestion
description: "Permanently delete a driveItem by using its ID."
```

Author: Done

doc_type: apiPageType
---

> # Permanently delete a DriveItem

Reviewer: ```suggestion
# Permanently delete a driveItem
```

Author: Done

Namespace: microsoft.graph

Permanently deletes a [DriveItem](../resources/driveitem.md) by using its ID.

> Note that deleting items using this method will be permanently deleted and won't go to recycleBin. Hence can't be restored.

Reviewer: ```suggestion
Note that if you delete items using this method, they will be permanently removed and won't be sent to the recycle bin. Therefore, they cannot be restored afterward.
```

Author: Done
